core/common/src/ChangedElements.ts

```diff
@@ -11,26 +11,57 @@ import { AxisAlignedBox3dProps } from "./geometry/Placement";
  * @module Entities
```

Comment:
TypeDoc requires `@packageDocumentation` to appear before any non-comments, hence the docs build failure.
Put it after the copyright header, before the imports.

Reply:
Thanks for the catch. Pushed